src/components/FlightBooking.tsx

```diff
@@ -1,70 +1,75 @@
-import { useCallback, useState } from 'react';
-
-import helpIcon from '../assets/help.svg';
-import plus from '../assets/plus.svg';
-import minus from '../assets/minus.svg';
-
+import { useState } from 'react';
 import styles from './FlightBooking.module.css';
 
-const MIN_PASSENGERS = 1;
 const MAX_PASSENGERS = 3;
 
 const FlightBooking = () => {
-  const [adultCount, setAdultCount] = useState(MIN_PASSENGERS);
-  const [statusMessage, setStatusMessage] = useState('');
-  const [showTooltip, setShowTooltip] = useState(false);
+  const [adultCount, setAdultCount] = useState(1);
+  const [announcement, setAnnouncement] = useState('');
+
+  const incrementCount = () => {
+    setAdultCount((prev) => {
+      if (prev === 3) {
```

Comment:
```suggestion
      if (prev === MAX_PASSENGERS) {
```
사소하지만 여기도 상수 값을 사용해도 좋을 것 같습니다🙃

Reply:
위의 피드백을 반영하면서 상수값도 같이 빼주었습니다~ 
https://github.com/woowacourse/a11y-airline/pull/155/commits/8ce4cfe3cf0aa42830e62e7480d9189aed3fb70b